### PR TITLE
add Introduction

### PR DESCRIPTION
**Pull request by nickhsine**

### CODE REVIEW

**Comment by hcchien** [comment]
lgtm

File: src/components/article/Introduction.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+$quoteMargin: 25px;
+$quoteTextColor: #666666;
+$quoteWidth: 30px;
+
+.introContainer {
+  position: relative;
+  margin-top: 90px;
+  margin-bottom: 90px;
+  font-size: 17px;
```
Comment (garfieldduck) [inline src/components/article/Introduction.scss]: put font-size as global scss variable (?

Comment (nickhsine) [inline src/components/article/Introduction.scss]: I use $font-size-base to calculate the font size.

**Comment by nickhsine** [comment]
https://github.com/twreporter/twreporter-react/issues/189

**Pull request merged**